packages/runtime/src/store/editor.ts

```diff
@@ -99,9 +99,9 @@ export class EditorStore {
 
   addFileOrFolder(file: FileDescriptor) {
     // when adding file or folder to empty folder, remove the empty folder from documents
-    const emptyFolder = this.files.get().find((f) => f.type === 'folder' && file.path.startsWith(f.path));
+    const emptyFolder = this.files.get().find((f) => file.path.startsWith(f.path));
 
-    if (emptyFolder && emptyFolder.type === 'folder') {
+    if (emptyFolder) {
```

Comment:
I removed this check for `f.type === 'folder'` because sometimes the folders were added as empty files first and I needed to delete those empty files (that were really folders).  I'm not sure if me removing this check will have unintended consequences though.